pages/groupmember.js

```diff
@@ -0,0 +1,297 @@
+import OrangeButton from "@/components/OrangeButton";
+import { faCopy } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useEffect, useState } from "react";
+import { keyframes, styled } from "styled-components";
+import { useSession } from "next-auth/react";
+import { useRouter } from "next/router";
+
+export default function GroupMember() {
+  const { data: session } = useSession();
+  const [groupSelection, setGroupSelection] = useState(false);
+  const randomstring = require("randomstring");
+  const [userData, setUserData] = useState();
+  const [activeGroupData, setActiveGroupData] = useState();
+  const [generatedLink, setGeneratedLink] = useState();
+  const [alluserGroups, setAllUserGroups] = useState();
+  const sendUserData = userData && userData;
```

Comment:
i don't get it :D

Reply:
this is an checkup if userData exists so i dont get an undefined, this is important for the code below